### PR TITLE
Firewolfgames patch 1

### PR DESCRIPTION
**Pull request by firewolfgames**

### CODE REVIEW

File: farming-simulator-nintendo-switch-edition/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (AcK77) [inline farming-simulator-nintendo-switch-edition/info.json]: Which one is the good file?

File: info.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (AcK77) [inline info.json]: Which one is the good file? and why this file and the screenshot are not in ```farming-simulator-nintendo-switch-edition``` folder? There is some Tab spacing missing too...

Comment (firewolfgames) [reply info.json]: This is the correct file, no idea how it got here :/

Comment (firewolfgames) [reply info.json]: Fixed it

**Comment by AcK77** [comment]
Could you fix the conflicts please, and fix the identation in the JSON file?

**Pull request closed without merging**